frontend/src/metabase/admin/settings/components/SettingsUpdatesForm/VersionUpdateNotice/VersionUpdateNotice.jsx

```diff
@@ -0,0 +1,169 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { t, jt } from "ttag";
+import { Flex, Box } from "grid-styled";
+
+import HostingInfoLink from "metabase/admin/settings/components/widgets/HostingInfoLink";
+import Icon from "metabase/components/Icon";
+import Text from "metabase/components/type/Text";
+
+import ExternalLink from "metabase/components/ExternalLink";
+import MetabaseSettings from "metabase/lib/settings";
+
+export default function VersionUpdateNotice() {
+  const currentVersion = formatVersion(MetabaseSettings.currentVersion());
+
+  if (MetabaseSettings.isHosted()) {
+    return <CloudCustomers currentVersion={currentVersion} />;
+  }
+
+  if (MetabaseSettings.versionIsLatest()) {
+    return <OnLatestVersion currentVersion={currentVersion} />;
+  }
+
+  if (MetabaseSettings.newVersionAvailable()) {
+    return <NewVersionAvailable currentVersion={currentVersion} />;
+  }
+
+  return <div>{t`No successful checks yet.`}</div>;
+}
+
+function CloudCustomers({ currentVersion }) {
+  return (
+    <div>
+      {jt`Metabase Cloud keeps your instance up-to-date. You're currently on version ${currentVersion}. Thanks for being a customer!`}
```

Comment:
Let's replace `jt` with `t` here and anywhere else where `t` works fine -- `jt` is for use when there are JSX components within the translation (I see that it's not your doing, but figure might as well tackle it now since it is straightforward to change).